package.json

```diff
@@ -591,6 +592,11 @@
                     "when": "view == dockerRegistries && viewItem =~ /DockerV2;Tag;/",
                     "group": "regs_tag_1_general@2"
                 },
+                {
+                    "command": "vscode-docker.registries.copyRemoteFullTag",
+                    "when": "view == dockerRegistries && viewItem =~ /DockerV2;Tag;/",
+                    "group": "regs_tag_1_general@2"
```

Comment:
This has the same value as for `copyImageDigest` so I'm not sure what order the UI will choose. We should adjust the other values. Personally I think "Copy Full Tag" should come before "Copy Image Digest", so that would mean keeping `regs_tag_1_general@2` for "Copy Full Tag", and reindexing "Copy Image Digest" and the others in the `regs_tag_1_general` section. Also, reordering the entries so that the order here in the manifest is the same as the order in the UI.